### PR TITLE
Update: Desktop version

### PR DESCRIPTION
**Pull request by blessedjasonmwanza**
## Update: Desktop version
In this update:
- Use CSS media queries to optimize for desktop screens
- Add navigation menu for desktop screens
- Optimize _Recent works_ and _Projects_ cards to adapt to desktop screens
- Optimize contact me form for desktop screens
- Fix linter errors and improve code semantics

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -13,6 +13,11 @@
     <nav class="mobile-nav-trigger d-flex flex-center">
```
Comment (SajjadAhmad14) [inline index.html:13]: Please add files and folders like `.vscode` to `.gitignore`.

**Review by SajjadAhmad14 — CHANGES_REQUESTED** [review]
## Required Changes ♻️
> "Change is inevitable. Change is constant. -Benjamin Disraeli"
## Hi Blessed Mwanza, 
 ### You have been doing a tremendous job so far :ok_hand:

## To Highlight 🎉
GitHub flow is followed. :heavy_check_mark:
README is professional. :heavy_check_mark:
## To Improve 🙏

- [x] Please check inline comments under the review

## Optional suggestions
_Comments with the [OPTIONAL] prefix are not a must to implement however, It's strongly recommended to work on them as they can make your code better._

## Happy coding!👏👏👏

#### Please leave any questions or comments in the PR if you have any doubts or unclarity!

**_Note: Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

File: index.html (hunk 1 above)
Comment (blessedjasonmwanza) [reply index.html:13]: Hello @SajjadAhmad14, 

> Thank you a lot for this insightful review. I have managed to do as guided and made the push.

_HappyCoding_

**Review by SajjadAhmad14 — APPROVED** [review]
# Status: Approved :heavy_check_mark: :heavy_check_mark: :heavy_check_mark:

### Hi Blessed Mwanza , 
## Congratulations 🎉 Thank you for implementing changes suggested previously 🙏

## Happy Coding 👏👏👏
#### Please leave any questions or comments in the PR if you have any doubts or unclarity!
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Pull request merged**